conf/prepush.sh

```diff
@@ -21,9 +21,9 @@ check_and_commit_updated_translations() {
 # lint, test, and build assets to update translations
 prepush() {
     echo "--------------------------------------------------------"
-    echo "Pre-build"
+    echo "Build bundles"
     echo "--------------------------------------------------------"
-    yarn pre-build || exit 1
+    yarn build:i18n || exit 1
```

Comment:
Do you need to clean just the i18n directories out first?

Reply:
Not necessary, since they are overwritten only if needed by checking contents